user-service/services/db.go

```diff
@@ -16,7 +16,7 @@ func GetDBConnection(v *viper.Viper, dbModels ...interface{}) (*gorm.DB, error)
 		v.GetString("database.password"),
```

Comment:
Move this file to a separate folder named initialization

Reply:
Done